src/goto-cc/gcc_cmdline.cpp

```diff
@@ -28,6 +28,7 @@ const char *goto_cc_options_with_separated_argument[]=
   "--native-linker",
   "--print-rejected-preprocessed-source",
   "--mangle-suffix",
+  "--object-bits",
```

Comment:
Shouldn't this be added to the other `cmdlinet`s as well?

Reply:
The original ticket referred to `goto-cc` only, fixing `goto-cc` only will get a fix ready for use sooner. Yes, the same problem and fix probably applies to the other entry points as well. The time consuming part of fixing the other entry points would be writing tests for each of them.

Reply:
There is no standard list of options in the base class. So it would need to be manually added and tested for each `cmdlinet` in the heirarchy.